.github/ISSUE_TEMPLATE/lib_change.yml

```diff
@@ -0,0 +1,57 @@
+name: "Library change"
+description: "Fix or improve issues with built-in type definitions like `lib.dom.d.ts`, `lib.es6.d.ts`, etc."
+title: "[LIB CHANGE] <description>"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ### Please fill in each section completely. Thank you!
+
+        Are you here for one of these commonly-requested lib changes?
+
+        * Object.keys - see https://stackoverflow.com/questions/55012174/
+        * Array methods - see https://github.com/microsoft/TypeScript/issues/36554
+        * `parseInt`, `parseFloat`, `isFinite`, `isNaN`, etc. - see https://github.com/microsoft/TypeScript/issues/4002
+
+
+        The DOM lib is maintained elsewhere and you can skip a step by filing issues/PRs for the DOM at that repo.
+        See https://github.com/microsoft/TypeScript-DOM-lib-generator
+  - type: textarea
+    id: configuration_check
+    attributes:
+      label: "⚙ Configuration Check"
+      description: |
+          If you're missing common new methods like Array.includes, you may have a misconfigured project.
+          Try setting `lib: "es2020"` and checking whether the type you want is present.
+
+          You can diagnose further by running `tsc` with `--listFilesOnly` or `--showConfig`.
+          Conversely, if you are seeing built-in methods you expect to *not* see, check your 'lib' setting
+
+          or review your dependencies for lib/reference directives that might be polluting
+          your global scope. This is common when using the 'node' type library. See https://github.com/microsoft/TypeScript/issues/40184
+
+          Replace the text below with the proper data.
+      value: "My compilation *target* is `ES2015` and my *lib* is `the default`."
```

Comment:
Nit, but maybe this should be two inputs or something; I thikn what we're asking for is for people to say what lib/target?

Reply:
He's saying there should be inputs for the specific value of `target` and `lib` instead of just asking the user to "replace the text below" a la old-fashioned templates

Reply:
Yes, I meant having two input boxes for each given that's the only two pieces of info we ask for here and they're required.